src/item/BoatType.php

```diff
@@ -33,7 +33,7 @@
  * @generate-registry-docblock
  *
  * @method static BoatType ACACIA()
- * @method static BoatType BAMBOO)
+ * @method static BoatType BAMBOO()
```

Comment:
This is supposed to be auto generated. Didn't you read the warning?

Reply:
I read it a bit to late xD Running "composer update-registry-annotations" add this commit